--- discourse.pill.sample	2015-04-24 14:01:08.726372097 +0000
+++ discourse.pill.hack	2015-04-24 14:01:39.126896770 +0000
@@ -17,23 +17,6 @@
   # getting this to work was a nightmare
   # bundle exec spawns a process totally messing with the demonize option
   # so we suck the environment out and set it up first
-  bootup_bundle = [ "#{ENV['HOME']}/.rvm/bin/rvm/bootup_bundle",
-                    "/usr/local/rvm/bin/rvm/bootup_bundle",
-                    `which bootup_bundle`.strip,
-                  ].each do |location|
-    if File.exist? location
-      break location
-    end
-  end
-  # XXX if none match, bootup_bundle is set to the array
-
-  if bootup_bundle
-    app.environment = `env -i BUNDLE_GEMFILE=#{rails_root}/Gemfile #{bootup_bundle} exec env`.lines.inject({}) do |env_hash,l|
-      kv = l.chomp.split('=',2)
-      env_hash[kv[0]] = kv[1]
-      env_hash
-    end
-  end
 
   app.environment ||= {}
 
