--- Gemfile	2015-09-09 13:55:21.975978766 +0000
+++ Gemfile.new	2015-09-09 13:58:23.519515204 +0000
@@ -96,7 +96,8 @@
 # for sidekiq web
 gem 'sinatra', require: false
 
-gem 'therubyracer'
+gem 'libv8', path: '/var/www/discourse/customgems/libv8'
+gem 'therubyracer', path: '/var/www/discourse/customgems/therubyracer'
 gem 'thin', require: false
 gem 'highline', require: false
 gem 'rack-protection' # security
